### DISCUSSION TITLE
TypeError: Cannot convert a BigInt value to a number #1450

### DISCUSSION THREAD

**Discussion by sayedex** [WalletConnect/walletconnect-monorepo#3878] [Developer Support]
![Screenshot - 2023-11-06T031001 222](https://github.com/WalletConnect/walletconnect-monorepo/assets/83809916/69ca01f5-73a5-4cc4-9e2f-b125ca094eb0)

i got this error when i try to install new v3; can anyone help?

**Comment by glitch-txs** [comment]
Can you share a public repository or code sandbox where we can reproduce this issue?

**Reply by sayedex** [reply]
here is my repo : https://github.com/sayedex/HexToys-Go-HEX1.git


now getting error Module not found: Can't resolve '@web3modal/wagmi/react' in '/home/sayed/Project/Hex/HexToys-Go-HEX/src/hooks'
